<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckIn</class>
 <widget class="QWidget" name="CheckIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生登记</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>171</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_stuNo">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_stuNo">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_stuNo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_stuName">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_stuName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_stuClass">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_stuClass">
       <property name="text">
        <string>班级：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="classComboBox_stuClass">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton_confirm">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_readStuName">
       <property name="text">
        <string>读文件</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
